.github/workflows/phar-creation.yml

```diff
@@ -4,22 +4,20 @@ name: "Building Require-Checker phar"
 
 on:
     pull_request:
```

Comment:
Because this step uses `SIGNING_SECRET_KEY`, it cannot (and shouldn't) really be used `on: [pull_request]`